### ISSUE TITLE
more options to quotes rule

### ISSUE THREAD

**Issue by kellyselden** [ember-template-lint/ember-template-lint#617]
I implemented the quotes rule the quickest and to satisfy the style I was using. It is now clear that there should be at least one more option for when you enter hbs syntax.

Some people prefer

```hbs
<foo bar="asdf" baz={{concat 'asdf' 'asdf'}} />
```

This is not possible with the current rule, where it is all or nothing double qoutes.

I'm not sure the api, but maybe something like

```js
'quotes': {
  outsideCurlies: 'double',
  insideCurlies: 'single'
}
```

**Comment by pieter-v** [comment]
This becomes more important, since prettier formats templates this way.

**Comment by robclancy** [comment]
Been open since 2019... how do we do this? When I enabled this rule I kinda expected it to just ignore the HTML quotes.

Also could the options be "htmlQuotes" and "hdsQuotes"?

**Comment by robclancy** [comment]
I think it's done https://github.com/ember-template-lint/ember-template-lint/pull/2754

**Comment by bmish** [comment]
Try it out in: https://github.com/ember-template-lint/ember-template-lint/releases/tag/v5.9.0
